### ISSUE TITLE
Question about Using GlobalForecast Code with Pre-cropped Data

### ISSUE THREAD

**Issue by taehokim97** [microsoft/ClimaX#33]
Hello? I have a question regarding the use of the ClimaX code.

I have collected data separately, and it has already been cropped for a specific region. I want to use this data for training with the GlobalForecast code.

I understand that the RegionForecast mentioned in the paper was trained by specifying regions for the entire dataset. Are there any specific considerations to keep in mind when using the GlobalForecast code without using the RegionForecast code?

For example, should I adjust the 'lat' input when calculating the loss, and so on?

Thank you.

**Comment by tung-nd** [comment]
I've not done this myself so I cannot say for sure, but these are a few things that come to my mind:
1) Make sure to use the same patch size as the pretrained checkpoint so you can load the entire pretrained model.
2) Interpolate the pretrained position embeddings if your data is of a different resolution. In the global forecasting module we also provide a function for doing this.
3) Input your latitude vector when finetuning the model since it'll be different from pretraning.

Besides this, finetuning ClimaX on your cropped data should be similar to global forecasting.

**Comment by taehokim97** [comment]
Thank you for your kind response.
